### PR TITLE
MAT-5398

### PR DESCRIPTION
**Pull request by ethankaplan**
## MADiE PR

Jira Ticket: [MAT-5398](https://jira.cms.gov/browse/MAT-5398)
(Optional) Related Tickets:
https://github.com/MeasureAuthoringTool/madie-measure/pull/316

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is in to the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/cqlLibraryLanding/CqlLibraryLanding.tsx [hunk 1]
```diff
@@ -92,7 +92,10 @@ function CqlLibraryLanding() {
         }}
       />
       <div className="measure-table">
-        <section tw="flex flex-row">
+        <section
+          tw="flex flex-row"
+          style={{ borderBottom: "1px solid #b0b0b0" }}
```
Comment (adongare) [inline src/components/cqlLibraryLanding/CqlLibraryLanding.tsx:97]: I think we need to handle it differently. there is already a line available if you see a couple of lines down after <Tabs> end. This is going to be a double line on it.
You will have to remove the the existing line or change its css. Also have to make sure that there is no gap between active tab indicator and this line.

**Review by adongare — APPROVED** [review]
